src/backoffice/backoffice.element.ts

```diff
@@ -83,45 +57,20 @@ export class UmbBackofficeElement extends UmbLitElement {
 	constructor() {
 		super();
 
+		// TODO: find a way this is possible outside this element.
 		this.provideContext(UMB_MODAL_CONTEXT_TOKEN, new UmbModalContext());
 		this.provideContext(UMB_NOTIFICATION_CONTEXT_TOKEN, new UmbNotificationContext());
-
-		// TODO: find a way this is possible outside this element. It needs to be possible to register stores in extensions
 		this.provideContext(UMB_CURRENT_USER_STORE_CONTEXT_TOKEN, new UmbCurrentUserStore());
```

Comment:
What about this store? Do we need to new it up here because it doesn't extend StoreBase?

Reply:
I left a couple of "special stores" and other globally registered contexts. Some if them we might come back to and register en the registry other context might "just" be registered as part of each areas entry point. 

I think we should do a second round of Backoffice Element cleanup. I just needed your packages/entry point work to be included to get the full overview.

Reply:
Sounds like awaiting an entry point extension point.